packages/x-charts/package.json

```diff
@@ -24,7 +25,7 @@
   ],
   "scripts": {
     "typescript": "tsc -p tsconfig.json",
-    "build": "yarn build:modern && yarn build:node && yarn build:stable && yarn build:types && yarn build:copy-files ",
+    "build": "pnpm build:modern && pnpm build:node && pnpm build:stable && pnpm build:types && pnpm build:copy-files ",
```

Comment:
Next step, use `nx` for orchestration 😄

Reply:
Oh yeah, we desperately need changes in that regard.
The CI times have become pointless. 🤷